@here/harp-mapview/lib/ElevationRangeSource.ts

```diff
@@ -58,3 +59,81 @@ export interface ElevationRangeSource {
      */
     ready(): boolean;
 }
+
+/**
+ * An [[ElevationRangeSource]] that delivers static values for the [[ElevationRange]], which are
+ * either setup from a [[DataSource]] or set in the constructor. Useful in situations where the
+ * geometry of the data is not (only) at ground level, where it may be clipped in certain
+ * situations.
+ */
+export class StaticElevationRangeSource implements ElevationRangeSource {
+    readonly m_tilingScheme: TilingScheme;
```

Comment:
`private` with getters and setter or remove the `m_` if they really should be public.

Reply:
Made them private.